docs/contribute.rst

```diff
@@ -267,19 +267,19 @@ Finally, if everything went well:
 * Merge the original release pull request on github
 * Tag the commit with ``x.y.0``, push tags ``git push --tags``
 * Create a new `github release`__ based on the tag above
-* Close the corresponding release milestone
-
-If the automation triggered by publishing the new github release
-was not successful, publish the fresh code to the `pypi`__
-repository manually::
+* Move the ``fedora`` branch to point to the release commit
```

Comment:
Or we could recommend to merge the release pull request from command line? This would prevent the extra rebase caused by the github button.

Reply:
Hm, that could be faster and easier

Reply:
Plus we get the correct hash in `tmt --version`:

```
tmt --version
tmt version: 1.8.0 (0577fe6)
```

The 0577fe6 is the old ref before merge button rebase.

Reply:
Very good point. So merge of the release PR has to be manual.

Reply:
PR updated with manual step